support-frontend/assets/components/productPage/productPageHero/productPageHero.jsx

```diff
@@ -72,41 +78,38 @@ const HeroHeading = ({
   </div>
 );
 
-const PaymentSelectionContainer = () => (
-  <div className="payment-selection-container">
-    <LeftMarginSection>
-      <PaymentSelection />
-    </LeftMarginSection>
-  </div>
-);
-
 const ProductPageHero = ({
-  overheading, heading, content, modifierClasses, children, appearance, hasCampaign, dailyEditionsVariant,
+  modifierClasses, children, appearance, showProductPageHeroHeader, ...props
```

Comment:
I don't think you need `showProductPageHeroHeader` in this list because it exists in props